CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+want to know what works well for projects in their contributor and governance
+areas. Looking at you graduated projects but we see the incubation project crew
+still working on great programs!
+- Be a part of our work in progress maintainers circle! [link to issue]
+
+Academics and open source enthusiasts that have completed research (or underway!)
+, studied, and/or created programs for other ecosystems, we would love to hear
+from you, too! Similar deal as above - watch out for our Issues, come hang out
+in slack, or we'll see you at a meeting!
+
+### Tips for filing issues
+//TODO add in comments about how to label once we figure that out
+
+### Pull Request Tips - Submitting and Reviewing
+- Talk to us via GitHub issue, mailing list, or meeting before you start on major
+work. If it’s typos and fixes to docs, please submit. 😀
+- Tap one of the chairs or working group facilitators in slack if you haven’t
+heard from us in 5 days.
+- When making changes to your PR, put them into a new commit (instead of rebasing
+  or amending) so that it's easier for reviewers to see just your new changes.
+- We will give you an opportunity to clean up your commits before merging or can
+squash them for you.
+- If a member asks you to "rebase" your PR, they're saying that a lot of docs/code
+ has changed, and that you need to update your branch to incorporate the latest
+ changes from master before we can merge you PR. You can use whatever you are
+ comfortable with, either merging master into your branch or rebasing your
+ branch on master.
+#### Reviewers:
+- We leave pull requests open for at least 48 hours, so that others have the
+chance to review/comment except in cases of the following:
+  - The majority of contributors in that working group have already reviewed
+  - Something urgent or small operational document change like a meeting time
+  - Spelling or grammar fixes
+
+## Your First Contribution to GitHub
+Open source communities use many different tools. If this is your first
+experience with GitHub - yay! There are a ton of great resources out there to
+educate on the UI and standard workflows. Everyone has been a beginner at some
+point; always feel free to ask for help. You can absolutely use the GitHub UI
+vs command line if that works best for your workflow.
+
+https://www.firsttimersonly.com/
+//TODO add more resources
+
+
+
+
+[CNCF’s code of conduct]: https://github.com/cncf/foundation/blob/master/code-of-conduct.md
+[communication channels]: https://github.com/cncf/sig-contributor-strategy/blob/master/README.md#communications
+[TOC]: https://www.cncf.io/people/technical-oversight-committee/
+[SIG]: https://github.com/cncf/toc/tree/master/sigs
+[README]: https://github.com/cncf/sig-contributor-strategy/blob/master/README.md
+[charter]: https://github.com/cncf/sig-contributor-strategy/blob/master/CHARTER.md
+[working groups]: https://github.com/cncf/sig-contributor-strategy/blob/master/README.md#working-groups
```

Comment:
For now we use the term subproject in the header, so let's make the link work and fix up in a follow-up PR on any other documents (readme/charter)

```suggestion
[working groups]: https://github.com/cncf/sig-contributor-strategy/blob/master/README.md#subprojects
```

Reply:
this is more of a cncf question. the sub groups of cncf sigs are using the term working groups and i think our group gets them confused because we say subprojects in k8s. @amye does this matter? can we use subprojects instead of working groups?

Reply:
https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs -- Working groups are preferred, per charter.